apps/web/modules/event-types/views/event-types-single-view.tsx

```diff
@@ -1,38 +1,11 @@
 "use client";
 
-import { EventType } from "@calcom/features/eventtypes/components/EventType";
-import { EventTypeAppDir } from "@calcom/features/eventtypes/components/EventTypeAppDir";
-import type { EventTypeSetupProps } from "@calcom/features/eventtypes/lib/types";
-
-/* eslint-disable @typescript-eslint/no-empty-function */
-// eslint-disable-next-line @calcom/eslint/deprecated-imports-next-router
-import { trpc } from "@calcom/trpc/react";
+import { EventType } from "@calcom/atoms/monorepo";
 
 import type { PageProps } from "@lib/event-types/[type]/getServerSideProps";
 
-const EventTypePageWrapper = (props: PageProps & { isAppDir?: boolean }) => {
-  const { data } = trpc.viewer.eventTypes.get.useQuery({ id: props.type });
-
-  if (!data) return null;
-
-  const eventType = data.eventType;
-
-  const { data: workflows } = trpc.viewer.workflows.getAllActiveWorkflows.useQuery({
-    eventType: {
-      id: props.type,
-      teamId: eventType.teamId,
-      userId: eventType.userId,
-      parent: eventType.parent,
-      metadata: eventType.metadata,
-    },
-  });
-
-  const propsData = {
-    ...(data as EventTypeSetupProps),
-    allActiveWorkflows: workflows,
-  };
-
-  return props.isAppDir ? <EventTypeAppDir {...propsData} /> : <EventType {...propsData} />;
+const EventTypePageWrapper = ({ type, ...rest }: PageProps) => {
```

Comment:
What is this file and component for if [EventTypeWebWrapper.tsx](https://github.com/calcom/cal.com/pull/16550/files#diff-86cbc78a9d170e869798e453a81849989ea411e3c4a36c856df0bccd3f01e8d5) has `const EventTypePageWrapper` defined on line 329 ? We have two EventTypePageWrappers.

Reply:
don't want to have stuff related to server side props inside of atoms, leaving this for wrapper component